handlers.py

```diff
@@ -61,40 +85,30 @@ async def process_feedback(message: types.Message, state: FSMContext):
     await state.finish()
 
 
-@dp.message_handler(lambda message: message.text == "Register")
-async def register(message: types.Message):
-    keyboard = types.ReplyKeyboardMarkup(resize_keyboard=True, one_time_keyboard=True)
-    button = types.KeyboardButton("Share a contact", request_contact=True)
-    keyboard.add(button)
-    await bot.send_message(chat_id=message.from_user.id, text="In order to use a bot, please share your contact!", reply_markup=keyboard)
-    await Form.register.set()
-
-    @dp.message_handler(content_types=['contact'], state=Form.register)
-    async def adding_to_db(message: types.Message, state: FSMContext):
-        try:
-            database.register(f"@{message.from_user.username}", message.from_user.first_name, message.from_user.id, message.contact['phone_number'])
-            await bot.send_message(chat_id=message.from_user.id, text="Thank you for the registration!")
-            await bot.send_message(USER_ID,
-                                   text=f"New user: {' '.join([message.from_user.first_name, f'@{message.from_user.username}', message.contact['phone_number']])}")
-        except sqlite3.IntegrityError:
-            await bot.send_message(chat_id=message.from_user.id, text="You've already been registered!")
-        await state.finish()
-
-
-@dp.message_handler(lambda message: message.text == "Track a person")
+@dp.message_handler(lambda message: message.text == "Track a person") # add state; index error if a user is not regitered
```

Comment:
by PEP-8 you need to add 2 spaces between the code and "#" when making comment inside the line

Reply:
solved